pkg/asset/installconfig/azure/client.go

```diff
@@ -248,12 +249,17 @@ func (c *Client) GetVirtualMachineSku(ctx context.Context, name, region string)
 			if !strings.EqualFold(name, *sku.Name) {
 				continue
 			}
+			return &sku, nil
+
+			// FIXME: I don't think the following block is needed anymore since
```

Comment:
I don't think we need it if you are confident the filter works. Haven't tried it myself but it looks like it should work.

Reply:
Right. I'll remove it if I don't see anything suspicious in the CI runs.